jdbc/pom.xml

```diff
@@ -265,6 +265,13 @@
                         <include>org/apache/hadoop/yarn/api/records/ApplicationId.class</include>
                       </includes>
                     </filter>
+                    <filter>
+                      <artifact>io.netty:netty-common</artifact>
+                      <includes>
+                        <include>io/netty/util/*</include>
```

Comment:
The issue is solved after the PR, should we shade these classes as well?

Reply:
HIVE-24138 reverted the `io.netty` shade classes. 

If we were to shade it again, I'm not sure if there will be a regression problem.

Reply:
Shade the `io.netty:netty-common  `https://github.com/apache/hive/pull/5981/commits/455b45215355b244bcd32e71640dde2c10db5bd7

I checked HIVE-24138, and it seems that HIVE-24138 is related [netty-buffer](https://mvnrepository.com/artifact/io.netty/netty-buffer) `io/netty/buffer/ArrowBuf`.

We just want to include `netty-common ` in this PR, so i think this change maybe safe.
@ayushtkn Could help to double check?